CMakeLists.txt

```diff
@@ -291,10 +291,14 @@ toolchain_cc_warning_error_implicit_int()
 # 'cmake -DEXTRA_CFLAGS="-Werror -Wno-deprecated-declarations" ..'
 include(cmake/extra_flags.cmake)
 
+#
+# mwdt doesn't have the following options
+if(COMPILER STREQUAL gcc)
```

Comment:
This check is not needed, as the zephyr_cc_option checks whether the given flag is supported by the compiler and in case it is not, it will not be added. Also, it may not only be gcc which needs these flags.

Reply:
Thanks for explanation, i will update this part